src/compiler/scala/tools/nsc/typechecker/RefChecks.scala

```diff
@@ -1145,11 +1145,12 @@ abstract class RefChecks extends Transform {
     }
 
     // scala/bug#6276 warn for trivial recursion, such as `def foo = foo` or `val bar: X = bar`, which come up more frequently than you might think.
+    // scala/bug#11614 warns for trivial recursion around `def x(): X = x`, covering a method with an apply
```

Comment:
This notation for the bug number is what should be used in the PR comment.

I see they already suggest on the next line that this should be an external lint, which today means scalafix. This change is helpful and benign, but it is still useful to add the scalafix and later delete this check.

Reply:
I updated the PR to remove the comment line.  That being said, I'm unsure of what the next steps are to handle this.  Should I create a PR in the [scalafix project](https://github.com/scalacenter/scalafix)?